test/hf_test.py

```diff
@@ -391,9 +417,8 @@ def test_case(self):
             if model_cls in [SwinForMaskedImageModeling, SwinForImageClassification, SwinModel]:
                 self.skipTest('Need to support SWIN models')
 
-            # TODO: support ViT models https://github.com/pytorch/PiPPy/issues/244
-            if model_cls in [ViTForMaskedImageModeling, ViTForImageClassification, ViTModel]:
-                self.skipTest('Need to support ViT models')
+            if model_cls in [ViTModel]:
```

Comment:
Instead of this please add `ViTModel` [here](https://github.com/pytorch/PiPPy/blob/ffa34589bc5e2981b1230d5a4b4625035eca1fb5/test/hf_test.py#L466)